### PR TITLE
Improve index and separate database section

### PR DESCRIPTION
**Pull request by jsanz**
Maybe it's easier to explore [the branch](https://github.com/CartoDB/carto-workshop/tree/improve-readme) to see the changes

### CODE REVIEW

**Comment by jsanz** [comment]
please @oriolbx @ernesmb take a look on the frontpage and section changes

File: 06-sdks/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-CARTO SDKs
-==========
+CARTO ENGINE I: development kits
+=======================================
 
 # Agenda
```
Comment (oriolbx) [inline 06-sdks/README.md]: In the README.md file, there are two different documents in lines 14 and 15 (they point to the python intro slides and python jupyter notebook) that have the same name (Python SDK Introduction), maybe it should be better to change one of them to `Python SDK Introduction slides` or `Python SDK Introduction examples`?

**Comment by oriolbx** [comment]
LGTM

**Comment by ernesmb** [comment]
looks good to me too

**Pull request merged**